tests/feature_extractor/test_main.py

```diff
@@ -89,15 +89,11 @@ def setUp(self):
 
         sp.check_call([
             "python", "-m", "neuron_morphology.feature_extractor", 
-            "--input_json", self.input_json_path, 
+            "--input_json", self.input_json_path,
             "--output_json", self.output_json_path
-        ])
+            ],
+            timeout=TIMEOUT)
 
-        stime = time.time()
```

Comment:
good change! You should probably remove the time import though

Reply:
👍